bot/exts/fun/cowsay.py

```diff
@@ -0,0 +1,56 @@
+from cowpy import cow
+from discord.ext import commands
+
+from bot.bot import Bot
+
+SUPPORTED_COWS = []
+
+for element in cow.get_cowacters():
+    # The cows in the list are too big for Discord or innapropriate
+    if element[0] not in ["turtle", "stegosaurus", "ghostbusters", "turkey", "mutilated", "kiss"]:
+        SUPPORTED_COWS.append(element[0] if element[0] != "default" else "cow")
+
+
+class Cowsay(commands.Cog):
+    """Cog for the Cowsay command."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command(
+        help=f"""
+        Generates a cowsay string and sends it. The available characters are:
+
+        {', '.join(SUPPORTED_COWS)}"""
+    )
+    async def cowsay(self, ctx: commands.Context, cowtype: str = "cow", message: str = None) -> None:
+        """Builds a cowsay string and sends it to discord."""
+        if message is None:
+            message = f"I'm a {cowtype}"
+
+        if cowtype == "cow":
+            cowtype = "default"
```

Comment:
Right now this isn't working if I don't include the cowtype after calling the command, since it is trying to pass the first word in the message as the cowtype. You might need to change the function definition to capture the rest of the message and modify the logic to handle the cowtype if none is passed.